util/util_windows.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2019-2020 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package util
+
+import (
+	"errors"
+	"syscall"
+	"time"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+var (
+	modkernel32 = windows.NewLazySystemDLL("kernel32.dll")
+
+	procGetSystemTimes = modkernel32.NewProc("GetSystemTimes")
+)
+
+/* misc */
+
+// RaiseRlimit increases the number of file descriptors we can have
+func RaiseRlimit(_ uint64) error {
+	return nil
+}
+
+// Getrusage gets file descriptors usage statistics
+func Getrusage(who int, rusage *syscall.Rusage) (err error) {
+	if rusage != nil {
+		*rusage = syscall.Rusage{}
+		err = nil
+	} else {
+		err = errors.New("invalid parameter")
+	}
+	return
+}
+
+// GetSystemTimes gets current OS kernel and usermode times
+func GetSystemTimes() (utime int64, stime int64) {
+	var idleTime, kernelTime, userTime syscall.Filetime
+
+	err := getSystemTimes(&idleTime, &kernelTime, &userTime)
+	if err == nil {
+		utime = filetimeToDuration(&userTime).Nanoseconds()
+		stime = filetimeToDuration(&kernelTime).Nanoseconds()
+	} else {
+		utime = 0
+		stime = 0
+	}
+	return
+}
+
+func filetimeToDuration(ft *syscall.Filetime) time.Duration {
+	n := int64(ft.HighDateTime)<<32 + int64(ft.LowDateTime) // in 100-nanosecond intervals
```

Comment:
Adding the `LowDataTime` is fine.. but you'd like to use uint64 here to avoid the negative sign.
The same holds true for the `HighDateTime`, but I don't know if we'll see that bit set any time soon.

Reply:
In the new code changed the retrieval of process times to `int64` like the nanoseconds used in Linux